pkg/workloads/console/controller.go

```diff
@@ -423,11 +424,30 @@ func (r *reconciler) buildJob(template *workloadsv1alpha1.ConsoleTemplate) *batc
 	// 1 pod per job.
 	backoffLimit := int32(0)
 
+	// jobName: Add console postfix and truncate name no longer than 49
+	// characters (63-14) (14 characters '-console-4mh4z')
```

Comment:
Entirely subjective, but, this comment might be clearer if it said something like:
> Ensure that the job name (after suffixing with `-console`) does not exceed 57 characters, to allow an additional 6 characters to appended when the job creates a pod without truncation of the `-console` suffix.